notification-channel-config-service-impl/src/main/java/org/hypertrace/notification/config/service/NotificationChannelConfigServiceImpl.java

```diff
@@ -30,12 +29,14 @@
 
   private final NotificationChannelStore notificationChannelStore;
   private final NotificationChannelConfigServiceRequestValidator validator;
+  private final Utils utils;
 
   public NotificationChannelConfigServiceImpl(
       Channel channel, ConfigChangeEventGenerator configChangeEventGenerator) {
     this.notificationChannelStore =
         new NotificationChannelStore(channel, configChangeEventGenerator);
     this.validator = new NotificationChannelConfigServiceRequestValidator();
+    this.utils = new Utils(this.notificationChannelStore);
```

Comment:
Are we not using guice? (If no, don't need to add it here, if yes, please use guice to instantiate instead).